src/utils.js

```diff
@@ -32,6 +32,17 @@ export function singleLineString(strings, ...vars) {
   }).join(' ').trim();
 }
 
+export function getVariable(context, name) {
```

Comment:
Ah ok - it's because whilst the utils is instrumented the rules aren't due to how they are run indirectly by eslint. We'll need to independently unit test this is isolation. More direct testing will be a good thing given util funcs like this will likely be used quite a lot. 

Mocking out the eslint APIs is fine if that makes it easier.